cogs/help.py

```diff
@@ -3,29 +3,52 @@
 import nextcord
 
 
-class HelpView(ui.View):
-    def __init__(self):
-        super().__init__(timeout=None)
-    # Nextcord help
-    @ui.button(label="Nextcord help", style=ButtonStyle.red, custom_id="help:nextcord")
-    async def nextcord_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Nextcord", interaction)
-    @ui.button(label="D.py libs help", style=ButtonStyle.gray, custom_id="help:dpylibs")
-    async def nextcord_lib_help(self, _, interaction: Interaction):
-        await self.create_help_thread("Discord.py libraries", interaction)
+class HelpView(nextcord.ui.View):
+    
+    @nextcord.ui.button(label='Nextcord Help?', style=nextcord.ButtonStyle.green)
+    async def nextcord_(self, button: nextcord.ui.Button, interaction: nextcord.Interaction):
+        view = Confirmation('Nextcord')
+        await interaction.response.send_message('Click "Yes!" to create the thread.', view=view, ephemeral=True)
```

Comment:
Possibly change it to something like "Are you really sure you want to make a help thread?"